filament/backend/include/backend/Platform.h

```diff
@@ -41,6 +41,24 @@ class UTILS_PUBLIC Platform {
     struct Fence {};
     struct Stream {};
 
+    /**
+     * The type of technique for stereoscopic rendering
+     */
```

Comment:
a note that it needs to match the materials

Reply:
done